src/pubsub/mod.rs

```diff
@@ -78,13 +82,13 @@ impl MessageValidator {
 
         // calculate and return the end to end latency
         let ts = u64::from_be_bytes([v[16], v[17], v[18], v[19], v[20], v[21], v[22], v[23]]);
```

Comment:
Not related to this patch but could this be made into a `v[16..24]`?

Reply:
Yes. Done.